src/internal-properties.js

```diff
@@ -402,7 +402,8 @@ const componentInternalProperties = {
     'resourceId',
     'height',
     'width',
-    'fit'
+    'fit',
+    'useDefaultIcon' // iOS|Android only
```

Comment:
@kpiascik The `useDefaultIcon` prop had been introduced for iOS|Android only. On Lumin it does nothing.

Reply:
I'm not sure @grozdanov would be a better person to ask.

Reply:
@pleszkiewicz, What is the default icon for iOS & Android ?

Reply:
Default icons are custom icons embedded within the project. So, if a dev does not import Lumin icons by putting 
```
"lumin-system-icons": "git+https://github.com/magic-script/system-icons"
```
in package.json, then iOS | Android will use default icons. Otherwise, it will use Lumin icons by default. The prop `useDefaultIcon` forces the iOS|Android to use default/custom icons even if there are Lumin icons linked to the project.